docs/content/2.frontier/2.installation/1.general.md

```diff
@@ -1,2 +1,3 @@
 # General Use
 
+When using the frontier cli tool it is prefered that when runing commands yarn is used over npm. When npm is used some frontier's mono repositoy features may not work as they should therefore yarn is needed. Using yarn also allows for parallel installation of the frontier tool to take place.
```

Comment:
Typos:
- "r" missing from "preferred" and "repository.
- "n" missing from "running".